source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -47,33 +49,43 @@ def delete_journeys(responses, request):
         logging.getLogger(__name__).info('filtering {} journeys'.format(nb_deleted))
 
 
-def filter_journeys(response_list, instance, request):
+def filter_journeys(responses, instance, request):
     """
     Filter by side effect the list of pb responses's journeys
 
-    first draft, we only remove the journeys with the same vjs
     """
-
-    # for clarity purpose we build a temporary list
-    journeys = [j for r in response_list for j in r.journeys]
-
+    is_debug = request.get('debug')
     #DEBUG
-    for j in journeys:
-        _debug_journey(j)
-
-    _filter_too_short_heavy_journeys(journeys, request)
+    if is_debug:
+        [_debug_journey(j) for j in get_qualified_journeys(responses)]
 
-    _filter_similar_vj_journeys(journeys, request)
-
-    _filter_too_long_waiting(journeys, request)
-
-    _filter_max_successive_physical_mode(journeys, instance, request)
+    min_nb_transfers = get_or_default(request, 'min_nb_transfers', 0)
 
-    _filter_min_transfers(journeys, instance, request)
+    # Note that we use the functools.partial to capture the arguments
```

Comment:
for python lovers: 
use lambda or partial? 
https://stackoverflow.com/q/3252228/1614576